Tasks/SonarQubePostTest/SonarQubeBuildBreaker.ps1

```diff
@@ -35,8 +35,8 @@ function FailBuildOnQualityGateStatus
     {        
         $dashboardUrl = GetTaskContextVariable "MSBuild.SonarQube.ProjectUri"
         
-        Write-Host "##vso[task.logissue type=error]The SonarQube quality gate associated with this build has failed. For more details see $dashboardUrl"
-        Write-Host "##vso[task.complete result=Failed;]"
+        Write-VstsTaskError "The SonarQube quality gate associated with this build has failed. For more details see $dashboardUrl"
```

Comment:
Can do, but be aware that the underlying tool (SonarQube) is EN only so there's not much point.